contracts/MemeFactory.sol

```diff
@@ -271,7 +330,7 @@ abstract contract MemeFactory {
         uint256 totalNativeTokenCommitted = memeSummon.nativeTokenContributed;
 
         // Check that the meme has been summoned
-        require(totalNativeTokenCommitted > 0, "Meme not yet summoned");
```

Comment:
This change will prevent hearing the redemption token.

Reply:
Check myself